pkgs/servers/http/openresty/default.nix

```diff
@@ -40,7 +40,8 @@ callPackage ../nginx/generic.nix args rec {
   configureFlags = [ "--with-http_postgres_module" ];
 
   postInstall = ''
-    ln -s $out/luajit/bin/luajit-2.1.0-beta3 $out/bin/luajit-openresty
+    ln -s $out/luajit/bin/luajit-2.1.ROLLING $out/bin/luajit-openresty
+    ln -sf $out/nginx/bin/nginx $out/bin/openresty
```

Comment:
Can you provide more context about addition of this line symlinking nginx to openresty, @lblasc?

Reply:
If I recall correctly, build will produce broken  `$out/bin/openresty` symlink, this line fixes that.

Reply:
Tried to build it and I confirm that the link is broken and this symlink update fixes it.
Original link points at bin/nginx:
```
❯ ls -la /nix/store/y1qy6lhscfcihbwkxfh9w3farsbigiyb-openresty-1.25.3.1/bin/
total 336
dr-xr-xr-x  11 root  wheel    352 Jan  1  1970 .
dr-xr-xr-x  12 root  wheel    384 Jan  1  1970 ..
lrwxr-xr-x   1 root  wheel     32 Jan  1  1970 luajit-openresty -> ../luajit/bin/luajit-2.1.ROLLING
-r-xr-xr-x   1 root  wheel  19232 Jan  1  1970 md2pod.pl
lrwxr-xr-x   1 root  wheel     18 Jan  1  1970 nginx -> ../nginx/bin/nginx
-r-xr-xr-x   1 root  wheel  16041 Jan  1  1970 nginx-xml2pod
lrwxr-xr-x   1 root  wheel      9 Jan  1  1970 openresty -> bin/nginx
-r-xr-xr-x   1 root  wheel  63697 Jan  1  1970 opm
-r-xr-xr-x   1 root  wheel  36900 Jan  1  1970 resty
-r-xr-xr-x   1 root  wheel  15004 Jan  1  1970 restydoc
-r-xr-xr-x   1 root  wheel   8920 Jan  1  1970 restydoc-index
```

Reply:
Thanks for the context, @lblasc, and thanks for chiming in and investigating, @kalekseev, much appreciated!